src/Task/SetupTask.php

```diff
@@ -37,11 +49,14 @@ class RealMeSetupTask extends BuildTask
     public function run($request)
     {
         try {
-            $this->service = Injector::inst()->get('RealMeService');
+            $this->service = Injector::inst()->get(RealMeService::class);
```

Comment:
We could do this with `private static $dependencies = ['Service' => '%$' . RealMeService::class];` and a getter/setter